setup.py

```diff
@@ -1,35 +1,35 @@
 # -*- coding: utf-8 -*-
 
-from setuptools import setup, find_packages
+from setuptools import setup
 
 setup(
-  name = 'markdown-jinja',
-  version = '0.1.0',
-  py_modules = ['markdown_jinja'],
-  description = 'Python Markdown extension which adds Jinja2 support',
-  author = 'Józef Sokołowski',
-  author_email = 'pypi@qzb.me',
-  url = 'https://github.com/qzb/markdown-jinja/',
-  download_url = 'https://github.com/qzb/markdown-jinja/archive/v0.1.0.tar.gz',
-  classifiers = [
-    'Development Status :: 3 - Alpha',
-    'License :: OSI Approved :: MIT License',
-    'Operating System :: OS Independent',
-    'Programming Language :: Python',
-    'Programming Language :: Python :: 2',
-    'Programming Language :: Python :: 2.7',
-    'Programming Language :: Python :: 3',
-    'Programming Language :: Python :: 3.4',
-    'Programming Language :: Python :: 3.5',
-    'Programming Language :: Python :: 3.6',
-    'Intended Audience :: Developers',
-    'Topic :: Communications :: Email :: Filters',
-    'Topic :: Internet :: WWW/HTTP :: Dynamic Content :: CGI Tools/Libraries',
-    'Topic :: Internet :: WWW/HTTP :: Site Management',
-    'Topic :: Software Development :: Documentation',
-    'Topic :: Software Development :: Libraries :: Python Modules',
-    'Topic :: Text Processing :: Filters',
-    'Topic :: Text Processing :: Markup :: HTML'
-  ],
-  install_requires = ['Markdown>=2.6', 'Jinja2>=2.10']
+    name='markdown-jinja',
+    version='1.0.0',
```

Comment:
As I understand semver correctly if the changes could break something or make backward-incompatible changes we must update the Major version. You can check this in [point 8](https://semver.org/#spec-item-8).

> Major version X (X.y.z | X > 0) MUST be incremented if any backwards incompatible changes are introduced to the public API. It MAY also include minor and patch level changes. Patch and minor versions MUST be reset to 0 when major version is incremented.

The bumping markdown version is a breaking change for me as we also changed some methods. What do you think?